src/sentry/services/hybrid_cloud/organizationmapping.py

```diff
@@ -0,0 +1,141 @@
+from abc import abstractmethod
+from dataclasses import dataclass, fields
+
+from django.db import IntegrityError, transaction
+from rest_framework import status
+
+from sentry.models.organizationmapping import OrganizationMapping
+from sentry.models.user import User
+from sentry.services.hybrid_cloud import InterfaceWithLifecycle, silo_mode_delegation
+from sentry.silo import SiloMode
+from sentry.silo.client import ControlSiloClient
+
+
+@dataclass(frozen=True, eq=True)
+class APIOrganizationMapping:
+    id: int = -1
+    organization_id: int = -1
+    slug: str = ""
+    region_name: str = ""
+
+
+class OrganizationMappingService(InterfaceWithLifecycle):
+    @abstractmethod
+    def create(
+        self,
```

Comment:
Minor style nit, but I've been using `*` here to force `kwds` on any service calls to allow for easier request schema upgrade.  Imagine that these params are one day gonna be some RPC object -- you'd want deprecated fields to be well supported, right?